src/gov/nist/core/LexerCore.java

```diff
@@ -51,6 +51,7 @@
     public static final int WHITESPACE = END + 1;
     public static final int DIGIT = END + 2;
     public static final int ALPHA = END + 3;
+    public static final int IPV6 = END + 4;
```

Comment:
yes, there is no conlission here, IPV6 at least greater than END (4096), the rest of individial token is smaller than 200.